build.gradle

```diff
@@ -300,6 +300,20 @@ allprojects {
     CacheRedirector.configure(project)
 }
 
+// --------------- Java 9 modularity ---------------
+
+// Currently the compilation of the module-info fails for both
+// kotlinx.coroutines.debug and kotlinx.coroutines.test
+// because they export packages that also exist in kotlinx.coroutines.core.
+// Note that only 1 module can export a package.
+def invalidModules = ["kotlinx-coroutines-debug", "kotlinx-coroutines-test"]
```

Comment:
`kotlinx-coroutines-test` has been fixed in 1.6.2 and no longer contains split package